File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/body/authentication/SendVerificationCodeBody.java
package com.lattis.ellipse.data.network.model.body.authentication;

import com.google.gson.annotations.SerializedName;

public class SendVerificationCodeBody {

//    @SerializedName("user_id")
//    public String userId;

    @SerializedName("user_id")
    public String user_id;

    @SerializedName("account_type")
    public String account_type;

    public SendVerificationCodeBody(String user_id, String account_type) {
        this.user_id = user_id;
        this.account_type = account_type;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/authentication/SendVerificationCodeBody.kt
package com.lattis.data.entity.body.authentication

import com.google.gson.annotations.SerializedName

class SendVerificationCodeBody(//    @SerializedName("user_id")
    //    public String userId;
    @field:SerializedName("user_id") var user_id: String?,
    @field:SerializedName("account_type") var account_type: String?
)